cypress/integration/gui/homepage.spec.js

```diff
@@ -16,16 +16,16 @@ describe("SLacktastic Homepage", () => {
 
 	it("contains three buttons", () => {
 		// assert:
-		cy.get(".buttonsContainer").eq(0).should("contain", "Admin");
-		cy.get(".buttonsContainer").first().should("contain", "Mentor");
-		cy.get(".buttonsContainer").last().should("contain", "Trainee");
+		cy.get(".buttonsContainer").eq(0).should("contain", "ADMIN");
```

Comment:
Hi Laura,
Thank you for updating the test.
The login page doesn't work by these new changes(ADMIN, MENTOR, TRAINEE).
Could you please return them back to the Admin, Mentor, and Trainee here and in login page ?
Thank you,